tools/tick-cluster.js

```
@@ -519,6 +525,60 @@ function terminateProc(count) {
     });
 }
 
+function restartProc(batchSize) {
+    if (batchSize === 0) {
+        return;
```

At all other places, `tick-cluster` silently ignore incorrect input